composer.json

```diff
@@ -18,12 +18,11 @@
     },
     "require-dev": {
         "zendframework/zend-eventmanager": "^2.5.1",
-        "zendframework/zend-inputfilter": "^2.5.1",
         "zendframework/zend-serializer": "^2.5.1",
         "zendframework/zend-servicemanager": "^2.5.1",
-        "zendframework/zend-filter": "^2.5.1",
         "phpunit/PHPUnit": "~4.0",
-        "squizlabs/php_codesniffer": "^2.0@dev"
+        "squizlabs/php_codesniffer": "^2.0@dev",
+        "zendframework/zend-inputfilter": "^2.5"
```

Comment:
zend-inputfilter is still a necessary dev requirement because test asset `\ZendTest\Hydrator\TestAsset\HydratorStrategyEntityA` utilizes code from the package.